### DISCUSSION TITLE
Don't load all locales

### DISCUSSION THREAD

**Discussion by desero** [elreco/vue-tailwind-datepicker#12] [Q&A]
First of all thank you for this great work. I'm wondering if there is a way to avoid loading all locales? You can see here that all the locales are requested on load: https://stackblitz.com/edit/vue-tailwind-datepicker

<img width="968" alt="image" src="https://user-images.githubusercontent.com/939367/198209690-b62692be-a1f6-4946-acdd-c17ec4c4d66d.png">

Thanks,
Mihai

**Comment by elreco** [comment]
@desero Hello! thank you for your message. 
Indeed, you are right. I don't know how to load it dynamically. I didn't found any documentation on how to do that with dayjs 😢 

Does anyone have an idea?
